frame_id: /map
planner_rate: 0.5
#map params
sizex: 600
sizey: 300
sizez: 100
origin_x: -35.025
origin_y: -12.025
origin_z: 0.0

#topic name params
goal_topic: /robot_goals
map_topic: /merged_map_update 
pose_topic: /robot_poses

#cost params
#objectmaxelev: 
#obsvalue: 
#freevalue: 
#unkvalue: 
#numangles:
#mindist: 

#planner params
#segbot/motionheight: 
#segbot/sensorheight:
#segbot/horizontalfov:
#segbot/verticalfov:
#segbot/detectionrange:
#segbot/circularsize:
#segbot/name:
#hexa/motionheight:
#hexa/sensorheight:
#hexa/horizontalfov:
#hexa/verticalfov:
#hexa/detectionrange:
#hexa/circularsize:
#hexa/name:
